### DISCUSSION TITLE
Axis/Ricom 3-10.5mm calibration file

### DISCUSSION THREAD

**Discussion by JC3** [lensfun/lensfun#1917] [Show and tell]
Here's a start for an Axis/Ricom 3-10.5mm DC iris IR lens, on an Axis M1125 camera, if it's useful to anybody:

```
<lens>
	<maker>Axis</maker>
	<model>01774-001 Ricom Varifocal 2MP 3-10.5mm F1.4 DC-iris IR</model>
	<mount>CS</mount>
	<cropfactor>7.2</cropfactor>
	<calibration>
		<distortion model="poly3" focal="3" k1="-.04959"/>
		<distortion model="poly3" focal="10.5" k1="-0.0051"/>
	</calibration>
</lens>
```

Notes:
- Because of the way this camera's zoom adjustment works, it's really hard to determine the exact focal length anywhere between the extremes (3 and 10.5mm).
- I'm assuming crop factor was relative to 35mm? In any case, this camera has a 6mm (1/3") sensor.
- I didn't do chromatic/vignette calibrations; the camera doesn't really have a noticeable vignette anyways, and chromatic aberrations would be difficult given the amount of post-processing the software on this camera does.
- Camera specs are [here](https://www.axis.com/products/axis-m1125/support#technical-specifications).
- Lens specs are ... around. The lens was common but is no longer in production as it's an older model.

**Comment by sarunasb** [comment]
Without focal length is an unknown, corrections' data would make little sense.

**Reply by JC3** [reply]
The 3mm and 10.5mm focal lengths are known. 

What I meant was; the tutorial gives advice to collect the extreme ends and also some in between, and I just couldn't collect any in between. That's all. :)
